README.adoc

```diff
@@ -198,16 +208,142 @@ curl http://localhost:9080/hello
 ```
 endif::[]
 
-=== Tearing down the Docker container
+After you are finished checking out the application, stop your container by running the following command:
+
+[role='command']
+```
+docker stop springBootContainer
+```
+
+ifndef::cloud-hosted[]
+== Optional: Faster container startup with InstantOn
+
+Open Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. In this section, you’ll learn how to build an InstantOn image that provides fast startup times. Your application can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features.
 
-To stop and remove your container, run the following commands:
+InstantOn requires a Linux system with kernel version 5.9 or greater. If your system does not meet this prerequisite, go to the <<running-the-application-on-open-liberty>> section. You can run the following command to check out the kernel version of your system:
+
+[role='command']
+```
+uname -r
+```
+
+The https://openliberty.io/docs/latest/reference/feature/crac.html[Coordinated Restore at Checkpoint^] (`crac`) feature enables applications to use the Liberty InstantOn implementation of the https://javadoc.io/doc/org.crac/crac/1.4.0/index.html[org.crac^] APIs. The Spring Framework hooks into the checkpoint and restore when the `org.crac` APIs are enabled.
```

Comment:
```suggestion
The https://openliberty.io/docs/latest/reference/feature/crac.html[Coordinated Restore at Checkpoint^] (`crac`) Liberty feature enables applications to use the Liberty InstantOn implementation of the https://javadoc.io/doc/org.crac/crac/1.4.0/index.html[org.crac^] APIs. When the `org.crac` APIs are enabled, checkpoint and restore can work with applications written using the Spring Framework.
```
Does this last sentence still work? It felt a little ungrammatical and jargon-y before.